src/main/resources/Species.json

```diff
@@ -0,0 +1,332 @@
+{
+  "ddis":{
+    "mart_url":"http:\/\/protists.ensembl.org\/biomart\/martservice",
```

Comment:
Is the escaping of the forward slashes required?

Reply:
They exist in the orthoinference code (since this file is ripped right from there). To be honest, a lot of these values aren't even used in AddLinks. Rather than get rid of them though, I talked with Solomon somewhere above about making this file generally accessible to all classes that use it. So for now, this file will stay here as is, but eventually it won't even be stored in this projects resources folder, but in something like `release-common-lib`.